### DISCUSSION TITLE
how can I change the default backgroundcolor/margin/etc. of ModalContent? tried to extend the theme but it didn't work...

### DISCUSSION THREAD

**Discussion by kingjulien1** [chakra-ui/chakra-ui#3857] [Q&A] [answered]
I already tried
``  ModalContent: {
    baseStyle: {
      background: "teal.100",
      my: 8,
    },
  },
``
with `extendTheme`, but it doesn't do anything. 
any suggestions?

**Comment by chr-ge** [comment, marked as answer]
This should work: 
```jsx
const theme = extendTheme({
    components: {
      Modal: {
        parts: ["dialog"],
        baseStyle: {
          dialog: {
            background: "teal.100",
            py: "8"
          }
        }
      }
    }
  });
```

**Reply by kingjulien1** [reply]
thanks, yea this did the trick. But what if I want to change the color dynamically, depending on light/dark theme?

**Reply by chr-ge** [reply]
```jsx
const theme = extendTheme({
  components: {
    Modal: {
      parts: ["dialog"],
      baseStyle: ({ colorMode }) => ({
        dialog: {
          background: colorMode === "dark" ? "blue.300" : "red.500",
          py: "8"
        }
      })
    }
  }
});
```

**Reply by kingjulien1** [reply]
is there somewhere a documentation on this stuff?

**Reply by chr-ge** [reply]
You can take a look at: [customize theme](https://chakra-ui.com/docs/theming/customize-theme#customizing-single-components) to see the light/dark theme example

**Reply by kingjulien1** [reply]
I read the documentation, there are however no detailed notations on specific use cases like this one (changing default bgcolor of modalcontent).

**Reply by kingjulien1** [reply]
for instance, now I want to limit the default `max` display of avatars in AvatarGroups to 2. Is there some place I can look up how to do this?

**Reply by chr-ge** [reply]
Since `max` isn't a style prop, you cannot set a default value in the theme, you would have to create a custom `AvatarGroup` in your app with `max={2}` to reuse.
